Apps/PackageTest/0.69.0/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "PackageTest",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+  },
+  "dependencies": {
+    "@babylonjs/core": "^5.6.1",
+    "@babylonjs/react-native": "^1.0.1",
+    "@babylonjs/react-native-iosandroid-0-69": "^1.0.1",
```

Comment:
Windows support will be added to PackageTest after the first 0-69 package is released.